accelerator/cli_info.h

```diff
@@ -63,6 +67,8 @@ static struct ta_cli_argument_s {
     {"ta_thread", optional_argument, NULL, TA_THREAD_COUNT_CLI, "TA executing thread"},
     {"iri_host", required_argument, NULL, IRI_HOST_CLI, "IRI listening host"},
     {"iri_port", required_argument, NULL, IRI_PORT_CLI, "IRI listening port"},
+    {"mqtt_host", required_argument, NULL, MQTT_HOST_CLI, "MQTT listening host"},
+    {"mqtt_topic_root", required_argument, NULL, MQTT_TOPIC_ROOT_CLI, "MQTT listening topic root"},
```

Comment:
Is it feasible to shorten the name of MQTT argument?

Reply:
how about change `mqtt_topic_root` to `mqtt_root`?